cypress/integration/badge-status.spec.ts

```diff
@@ -0,0 +1,53 @@
+describe("Badge Status", () => {
```

Comment:
Great job at implementing this test. I'd probably move this to the `project-list.spec` file. Seems to belong there from my perspective

Reply:
I have a question, Should I need to create a new PR for this? Thanks!